packages/qwik/src/core/v2/client/vnode-diff.ts

```diff
@@ -621,9 +623,66 @@ export const vnode_diff = (
       element.setAttribute('class', scopedStyleIdPrefix);
     }
 
+    vnode_insertBefore(
+      journal,
+      container.document,
+      vParent as ElementVNode,
+      vNewNode as ElementVNode,
+      vCurrent
+    );
+
     return needsQDispatchEventPatch;
   }
 
+  function createElementWithNamespace(tag: string): Element {
+    const domParentVNode = vnode_getDomParentVNode(vParent);
+
+    const parentIsDefaultNamespace = domParentVNode
+      ? vnode_getElementName(domParentVNode) && vnode_isDefaultNamespace(domParentVNode)
```

Comment:
`vnode_getElementName(domParentVNode)` this will read a "real" element name if we don't have it, but I don't have idea how to do it another way